doc/src/hexdoc/_templates/macros/recipes.html.jinja

```diff
@@ -113,4 +113,105 @@
       {% endfor %}
     </div>
   </details>
-{%- endmacro %}
\ No newline at end of file
+{%- endmacro %}
+
+{% macro render_mindcipe(recipe) -%}
```

Comment:
cute name! unfortunately i do want to lean more towards clarity over whimsy. maybe something like `brainsweep_recipe()` or `flay_mind_recipe()`? unsure